src/components/Form/FormTypes.tsx

```diff
@@ -26,4 +26,6 @@ export interface FormProps {
 	buttons?: ButtonProps[];
 	tooltipInfo?: string;
 	showActive?: boolean;
+	setTopComponentDrawerHeight?: Dispatch<SetStateAction<string>>;

```

Comment:
I don't like the idea of passing these refs / setStates to the Form because it's not its responsibility and also this can cause issues with other developers passing refs. I'm thinking on another solution that might involve context / reducers, that way the only components responsible are the ones involved (AdvancedSelection, TopComponent and FormLayout)